js/src/product-feed/review-request/review-request-modal.js

```diff
@@ -87,15 +87,17 @@ const ReviewRequestModal = ( {
 				</p>
 			</Notice>
 			<ReviewRequestIssues issues={ issues } />
-			<CheckboxControl
-				className="gla-review-request-modal__checkbox"
-				label={ __(
-					'I have resolved all the issue(s) listed above.',
-					'google-listings-and-ads'
-				) }
-				checked={ checkBoxChecked }
-				onChange={ handleCheckboxChange }
-			/>
+			{ issues.length > 0 && (
```

Comment:
❓

Is it not better to check if there are issues on the top of this file? If issues.length === 0 do not render this modal? 

Or maybe is even better if we do it just [before this line](https://github.com/woocommerce/google-listings-and-ads/blob/6f863dcf3ef87074ac973f11cc51bfd6e9261d36/js/src/product-feed/review-request/index.js#L58) and we do not render the `<ReviewRequestModal/>` component at all.

Reply:
Let me explain the rationale behind this: 

Before this PR we actually hide the modal if there were no issues:

`if ( ! issues.length || ! isActive ) {` 

Right now we still render the modal if there is no issues. That's pretty much defensive since in case the account shows the request button it means there are issues but in the worst case scenario that for some reason the status was DISAPPROVED but without providing any issue (maybe a Google API issue or a bug on my side) it'd still show the modal allowing the user to request the review. 

If we avoid rendering the modal if no issues, in case this rare error happens it will show the Request Review button but it will never open the modal since there are no issues.